supervisor.conf

```diff
@@ -1,11 +1,12 @@
 [program:run]
-command = /opt/odo/bin/setup-and-run
+command = /opt/odo/bin/run
```

Comment:
with this change, the `setup-and-run` is no longer used. Shouldn't we remove that file?

Reply:
Yes! Done!